template/source.c.erb

```diff
@@ -97,8 +103,7 @@ source_initialize(bt_self_component_source *self_component_source,
   /* Register User Callbacks */
   btx_register_usr_callbacks((void *)common_data);
   /* Call initialize_processing*/
-  btx_call_callbacks_initialize_component((void *)common_data,
-                                          &common_data->usr_data);
+  btx_call_callbacks_initialize_component(common_data, &common_data->usr_data);
```

Comment:
Why was the (void *) cast removed for common_data?. Same in sink and filter.

Reply:
Not required, `btx_call_callbacks_initialize_component` doesn't take a void pointer. Not sure why